save_princess_2/test/bot_test.rb

```diff
@@ -1,11 +1,35 @@
 require 'minitest/autorun'
 require 'minitest/pride'
-require './lib/display_path'
+require './lib/bot'
 
 class DisplayPathTest < Minitest::Test
   def setup
     @bot = Bot.new
   end
 
-  
+  def test_it_exists
+    assert_instance_of Bot, @bot
+  end
+
+  def test_it_can_find_princess
+    grid = ['-----', '-----', 'p--m-', '-----', '-----']
+    assert_equal [2, 0], @bot.find_princess(grid)
+
+    grid2 = ['----p', '-----', '--m--', '-----', '-----']
+    assert_equal [0, 4], @bot.find_princess(grid2)
```

Comment:
Could add edge case test checking for no princess. But that is not likely to be a scenario with the given conditions